presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -232,14 +322,10 @@ public void testRefreshing()
         copy(new File(getResourcePath("security-config-file-with-unknown-rules.json")), configFile);
         sleep(2);
 
-        assertThatThrownBy(() -> accessControl.checkCanCreateView(alice, aliceView))
-                .isInstanceOf(IllegalArgumentException.class)
-                .hasMessageStartingWith("Invalid JSON file");
+        assertDeniedWithIllegalArgumentException(() -> accessControl.checkCanCreateView(alice, aliceView), INVALID_JSON_ERR_MESSAGE);
```

Comment:
I don't think there is a value in this refactor. Let's leave it as it was.

Reply:
I have reverted this back.